vanilla-todo/main.js

```diff
@@ -0,0 +1,95 @@
+// Import elements
+const toDoForm = document.getElementById("todo-form"),
+  toDoTitle = toDoForm.querySelector("h3"),
+  toDoInput = toDoForm.querySelector("input"),
+  toDoList = document.getElementById("todo-list");
```

Comment:
```suggestion
const $toDoForm = document.getElementById("todo-form"),
  $toDoTitle = toDoForm.querySelector("h3"),
  $toDoInput = toDoForm.querySelector("input"),
  $toDoList = document.getElementById("todo-list");
```
보통 element에는 이렇게 `$`를 붙인답니다. element와 일반 변수를 구분하기 위한 용도라고 생각해주세요~